.gitlab-ci.yml

```diff
@@ -304,11 +304,15 @@ Integration:
         INTERNAL_NETWORK: ["true"]
 
 .API_TESTS: &API_TESTS
-  TARGET:
-    - azure
+  IMAGE_TYPE:
     - aws
+    - azure
+    - edge-commit
+    - edge-container
     - gcp
-    - aws.s3
+    - image-installer
+    - guest-image
+    - vsphere
```

Comment:
The point that Ondřej made is still valid I think, isn't it enough to just test 1 aws.s3 target?